### ISSUE TITLE
What is the validation dataset?

### ISSUE THREAD

**Issue by sourachakra** [siyueyu/DCFM#3]
How to regulate model training? Do you use any validation dataset to ensure that the trained model does not overfits?

**Comment by siyueyu** [comment]
@sourachakra  I used the CoCA for model selection, which is similar to the based network GCoNet

**Comment by sourachakra** [comment]
Thank you for your reply. Also, did you use the IoU loss metric for validation or the other metrics like E-measure, S-measure, F-measure and the MAE (used for testing) for that purpose? I am actually trying to reproduce the paper model as close as possible. Therefore, wondering about the details. Thanks.

**Comment by siyueyu** [comment]
Actually, I only focuses on F-measure. In my experiment, I only tried the models after 190 epochs.
